jenkinsci/blueocean-plugin
# JENKINS-50604: modify Bitbucket create test to follow an entire create flow

# Description

See [JENKINS-50604](https://issues.jenkins-ci.org/browse/JENKINS-50604), which was inspired by [JENKINS-50572](https://issues.jenkins-ci.org/browse/JENKINS-50572). This change extends `BitbucketServerTest#testCreationNoJenkinsfile` to walk through an entire Pipeline creation flow, including saving the Jenkinsfile. Previous to this PR, the test was stopping once the Create button had been clicked, but there was never any pipeline being saved. It is my hope that this will help catch regressions like [JENKINS-50572](https://issues.jenkins-ci.org/browse/JENKINS-50572).

To test: 
- Start up the acceptance tests in your preferred way. 
- Run `mvn test -Dtest=BitbucketServerTest`
- As of this writing, because of [JENKINS-50572](https://issues.jenkins-ci.org/browse/JENKINS-50572), this test will usually fail. This is expected until 50572 is triaged/fixed.

# Submitter checklist

# Reviewer checklist
